### DISCUSSION TITLE
Loss curves

### DISCUSSION THREAD

**Discussion by Aymen08** [coqui-ai/TTS#992] [Qs for Live Q&A] [answered]
Hi,
when training a tts model, which script/command we should use to plot loss curves?

**Comment by Lizh-TW** [comment]
You could using tensorboard.

**Reply by Aymen08** [reply]
Thank you

**Comment by WeberJulian** [comment, marked as answer]
cd into the output folder (you should see a `events.out.tfevents.1638435594....` file) and then `tensorboard --logdir . --bind_all`

**Reply by Aymen08** [reply]
Thank you 
It is clear now
